Cargador.cpp

```diff
@@ -0,0 +1,53 @@
+Cargador::Cargador()
+{
+
+}
+
+void Cargador::cargadorDeABB(ABB<Diccionario> &arbolAeropuertos , ifstream archivo)
+{
+    //char[3] o string para el codigo IATA?
+    string nombre ,ciudad ,pais,codigoIATA ;
+    double superficie;
+    int cantidadTerminales,destinosNacionales,destinosInternacionales ;
+
+
+    if(!archivo.fail())
+    {
+        while(!archivo.eof())
+        {
+            //Puntero a los datos del aeropuerto
+            Aeropuerto *pAeropuerto ;
+
+            archivo >> codigoIATA;
+            archivo >> nombre;
+            archivo >> ciudad;
+            archivo >> pais;
+            archivo >> superficie;
+            archivo >> cantidadTerminales;
+            archivo >> destinosNacionales;
+            archivo >> destinosInternacionales;
```

Comment:
Se pueden acortar todas estas lineas así:
```suggestion
            archivo >> codigoIATA  >> nombre >> ciudad >> pais >> superficie >> cantidadTerminales >> destinosNacionales >> destinosInternacionales;
```

Reply:
Lo dejé asi por cuestiones mas esteticas del codigo , pero lo cambiamos, como quierann

Reply:
Esta perfecto, dejalo así. Era un comentario nomás por si no sabías